### ISSUE TITLE
No sub-pixel-antialias (LCD text) on HTML-Canvas in Linux build

### ISSUE THREAD

**Issue by Z3TA** [nwjs/nw.js#4325]
On the linux build of nw.js there are grayscale antialias in HTML Canvas, where it should be sub-pixel (LCD text) antialias when passing {alpha: false}: ctx = canvas.getContext("2d", {alpha: false});

I am developing a text editor and want as crisp text as possible! It works fine with Windows and text outside the canvas in Linux.

**Update**: The Canvas _sometimes_ renders with LCD-Text! I think it is some type of internal optimization in Chromium. Also, you can't disable CLD-Text using --disable-lcd-text in Linux. But in Windows you can, and it always renders text with LCD sub-pixel-antialas (LCD-text) in Windows (if you have it on).

Basically I want on option to either force LCD-Text on on or off. The current inconsistent behaviour is annoying.
## <bountysource-plugin>

Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/30295042-no-sub-pixel-antialias-lcd-text-on-html-canvas-in-linux-build?utm_campaign=plugin&utm_content=tracker%2F51989&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F51989&utm_medium=issues&utm_source=github).
</bountysource-plugin

**Comment by Christywl** [comment]
@Z3TA , is this issue still reproduced on the latest nwjs build? If so, could you please provide a sample and steps to reproduce it?

**Comment by Z3TA** [comment]
@Christywl , I just tested with "0.22.3" on Linux. And sub pixel anti-alias seem to work in the Canvas!
